### ISSUE TITLE
JSX in switch/case statement

### ISSUE THREAD

**Issue by kidkkr** [palantir/tslint-react#170]
I have found that 

```js
switch(someCondition) {
  case A: return <A />;
  case B: return <B />;
  default: <div />;
}
```

warns nothing for standalone JSX statement.
is there a rule for this?

**Comment by adidahiya** [comment]
What warning are you expecting?

**Comment by kidkkr** [comment]
Hmm... I thought that It should return value in default case
But It looks okay because It could return value somewhere else or return undefined.
Maybe floating JSX (which `React.createElement('...')` is not assigned or returned) should be warned.

**Comment by adidahiya** [comment]
Closing due to deprecation, see #210
